images/macos/templates/macOS-13.arm64.anka.pkr.hcl

```diff
@@ -192,7 +202,9 @@ build {
     script = "./provision/core/xcode.ps1"
     environment_vars = [
       "XCODE_INSTALL_USER=${var.xcode_install_user}",
-      "XCODE_INSTALL_PASSWORD=${var.xcode_install_password}"
+      "XCODE_INSTALL_PASSWORD=${var.xcode_install_password}",
```

Comment:
I think `XCODE_INSTALL_USER` and `XCODE_INSTALL_PASSWORD` variables are not needed anymore. Could you please remove them from template?

Reply:
done